index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
+            <img class="menubars" src="menu.png" alt="Menu"> 
```

Comment:
- [ ] Also make sure the `nav` tag surround only the `hamburger menu` for semantic reasons.